addons/website/static/src/core/menu_callback_service.js

```diff
@@ -0,0 +1,14 @@
+import { registry } from "@web/core/registry";
```

Comment:
Actually I agree with Géry, `website_headers_service` may be better than "menu".

Reply:
`website_menus_service` would be better because it's linked to the menus of FaqHorizontal & TableOfContent.
It is also linked to the header, but it's a menu/title behavior.